src/tasks/recursionReverseString.js

```diff
@@ -0,0 +1,7 @@
+export const reverseString = (str) => {
+  if (str === "") {
+    return str;
+  } else {
+    return reverseString(str.slice(1)) + str[0];
```

Comment:
You use an early return approach. You can get rid of   'else' block. Analyze code and remove useless 'else'